docs/source/_static/my-styles.css

```diff
@@ -0,0 +1,77 @@
+@font-face {
+    font-family: "Computer Modern";
+    src: url("./cmunrm.ttf");
+}
+
+
+@font-face {
+    font-family: "Computer Modern";
+    src: url("./cmunbx.ttf");
+    font-weight: bold;
+  }
+
+@font-face {
+    font-family: "Computer Modern";
+    src: url("./cmunti.ttf");
+    font-style: italic;
+}
+
+style {
+    bar
+}
+
+
+body {
+    font-family: "Computer Modern";
+}
+
+h1 {
+    font-family: "Computer Modern";
```

Comment:
Should change this assignment of the same font family to 

```css
body,
h1,
h2,
<The rest of the tags> {
    font-family: "Computer Modern";
}
```

Reply:
corrected